pages/search.js

```diff
@@ -340,6 +340,9 @@ class SearchPage extends React.Component {
               />
             </Hide>
             <Hide md lg>
+              <FilterLabel htmlFor="profile-filter-type">
```

Comment:
`htmlFor` must match the input ID. No blocker for me to merge once this is fixed

```suggestion
              <FilterLabel htmlFor="collective-filter-type">
```